config/routes.rb

```diff
@@ -1,11 +1,12 @@
 IntergalacticBattleship::Application.routes.draw do
+  devise_for :users
   root "strikes#index"
   resources :strikes, only: [:new, :create]
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
-  root 'static_pages#index'
```

Comment:
Yeah we shouldn't have two root pages, so removing one is good, but I think we should remove the other one to keep the tests passing.  So keep:

```
root 'static_pages#index'
```

And remove:

```
root "strikes#index"
```

And probably add it into the strikes section.

```
resources :strikes, only: [:new, :create]
```